relayer/src/worker/packet.rs

```diff
@@ -110,10 +110,13 @@ impl PacketWorker {
                     // once at start, and _forced_ at predefined block intervals.
                     let force_packet_clearing = self.clear_packets_interval != 0
                         && height.revision_height % self.clear_packets_interval == 0;
+
                     link.a_to_b
-                        .schedule_packet_clearing(height, force_packet_clearing)
+                        .schedule_packet_clearing(Some(height), force_packet_clearing)
                 }
 
+                WorkerCmd::ClearPendingPackets => link.a_to_b.schedule_packet_clearing(None, true),
```

Comment:
Is okay to clear pending packets up to the latest height here? ie. whenever we detect that the WS subscription has been closed and that we may have missed events.

Reply:
I think so!